source/Patches/TaskPatches.cs

```diff
@@ -28,6 +28,7 @@ private static bool Prefix(GameData __instance)
                             if (playerInfo.Tasks.ToArray()[j].Complete) __instance.CompletedTasks++;
                         }
                 }
+                Patches.EndGameSummary.UpdatePlayerInfo();
```

Comment:
Is there a reason we can't just do this right before we need it? If so, would love to have that documented in the code. Comments that explain "why" we do something, especially something weird or counter-intuitive, are very important.

Reply:
i will try again putting it just before the end game, but it wasnt working before, so i had to put it back in TaskPatches,

ive put it back at the place i did before (AmongUsClient_OnGameEnd.PostFix) and it works there again, so i will remove it from TaskPatches.

regarding comments, i normally put in tonnes of comments, i hate uncommented code, but i saw no comments in the code in other PRs so figured the comments were not wanted and removed them before i committed.